test/controllers/notifications_controller_test.rb

```diff
@@ -12,8 +12,8 @@ class NotificationsControllerTest < ActionController::TestCase
   end
 
   test "should assign unapproved sign slides" do
-    users(:two).signs << signs(:one)
-    signs(:one).slides << slides(:two)
+    users(:two).signs << signs(:default)
```

Comment:
No I haven't made any changes to the user fixture yet, I tried removing `two` from the slides fixture.